### PR TITLE
Migrate Adding Patterns Test to Playwright

### PR DESCRIPTION
**Pull request by pooja-muchandikar**
## What?

Part of https://github.com/WordPress/gutenberg/issues/38851.
Migrate[ adding-patterns.test.js ](https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-tests/specs/editor/various/adding-patterns.test.js) to its Playwright version.
## Why?

Part of https://github.com/WordPress/gutenberg/issues/38851.
## How?

See[ MIGRATION.md](https://github.com/WordPress/gutenberg/blob/trunk/test/e2e/MIGRATION.md) for migration steps.
## Testing Instructions

Run` npm run test:e2e:playwright test/e2e/specs/editor/various/adding-patterns.spec.js`

### CODE REVIEW

File: test/e2e/specs/editor/various/adding-patterns.spec.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+			'Social links with a shared background color'
+		);
+
+		await page.click(
+			'role=option[name="Social links with a shared background color"i]'
+		);
+		await expect.poll( editor.getBlocks ).toMatchObject( [
+			{
+				name: 'core/social-links',
+				attributes: { className: 'has-icon-color' },
```
Comment (kevin940726) [inline test/e2e/specs/editor/various/adding-patterns.spec.js]: Nit: I don't think we need this. I think we only want to check if it's a `social-links` block.

```suggestion
```

**Review by kevin940726 — APPROVED** [review]
Great work! Thanks!

File: test/e2e/specs/editor/various/adding-patterns.spec.js (hunk 1 above)
Comment (pooja-muchandikar) [reply test/e2e/specs/editor/various/adding-patterns.spec.js]: Addressed the feedback!

**Comment by pooja-muchandikar** [comment]
Hi @kevin940726,

Thanks for the suggestions, I have addressed those but seems like the CI is failing but its not due to my test case, but its failing for `template-part.spec.js` test case. Could you please take a look?

image

Thanks!

**Pull request merged**